modules/apps/commerce/commerce-test/src/testIntegration/java/com/liferay/commerce/price/test/CommerceProductPriceCalculationWithTaxV2Test.java

```diff
@@ -185,14 +186,34 @@ public void testGetGrossProductPriceFromGrossPriceList() throws Exception {
 
 	@Test
 	public void testGetGrossProductPriceFromNetPriceList() throws Exception {
+		CommerceChannel commerceTaxIncludedChannel =
```

Comment:
@igorbeslic name should be like this.
```suggestion
		CommerceChannel taxIncludedCommerceChannel =
```